src/Ruleset.php

```diff
@@ -892,6 +892,13 @@ private function expandSniffDirectory($directory, $depth=0)
      */
     private function expandRulesetReference($ref, $rulesetDir, $depth=0)
     {
+        // Naming an (external) standard "Internal" is deprecated.
+        if (strtolower($ref) === 'internal') {
+            $message  = 'The name "Internal" is reserved for internal use. A PHP_CodeSniffer standard should not be called "Internal".'.PHP_EOL;
```

Comment:
I tried to remove one use of "internal" from the message to avoid repeating it three times. I came up with the following suggestion:

```suggestion
            $message  = 'The name "Internal" is reserved for internal use. A PHP_CodeSniffer standard should not it.'.PHP_EOL;
```

What do you think? This is just a nitpick. Feel free to ignore it.

Reply:
@rodrigoprimo The suggested change is a grammatical error. Want to have another look ?

Reply:
That is a fair point.